content/news/2023-03-04-bevy-0.10/index.md

```diff
@@ -0,0 +1,1882 @@
+
+## `EntityCommand`s
+
+<div class="release-feature-authors">authors: @targrub</div>
+
+[`Commands`] are "deferred ECS" operations. They enable developers to define custom ECS operations that are applied after a parallel system has finished running. Many [`Commands`] ran on individual entities, but this pattern was a bit cumbersome:
+
+```rust
+struct MyCustomCommand(Entity);
+
+impl Command for MyCustomCommand {
+    fn write(self, world: &mut World) {
+        // do something with the entity at self.0
+    }
+}
+
+let id = commands.spawn(SpriteBundle::default()).id();
+commmands.add(MyCustomCommand(id));
+```
+
+To solve this, in **Bevy 0.10** we added the [`EntityCommand`] trait. This allows the command to be ergonomically applied to spawned entities:
+
+```rust
+struct MyCustomCommand;
+
+impl EntityCommand for MyCustomCommand {
+    fn write(self, id: Entity, world: &mut World) {
+        // do something with the given entity id
+    }
+}
+
+commands.spawn(SpriteBundle::default()).add(MyCustomCommand);
+```
+
+[`EntityCommand`]: https://docs.rs/bevy/0.10.0/bevy/ecs/system/trait.EntityCommand.html
+[`Commands`]: https://docs.rs/bevy/0.10.0/bevy/ecs/system/struct.Commands.html
+
+## Pixel Perfect Example
+
+<div class="release-feature-authors">authors: @Ian-Yy</div>
+
+We now have a new ["pixel perfect" example](https://github.com/bevyengine/bevy/blob/v0.10.0/examples/2d/pixel_perfect.rs) that illustrates how to set up pixel-perfect sprites. It uses a cute new Bevy logo sprite!
+
+![pixel perfect](pixel_perfect.png)
+
+## UI Text Layout Example
+
+<div class="release-feature-authors">authors: @ickshonpe</div>
+
+We've added a nice ["text layout" example](https://github.com/bevyengine/bevy/blob/v0.10.0/examples/ui/text_layout.rs) that illustrates the various Bevy UI text layout settings:
+
+![text layout](text_layout.png)
+
+## CI Improvements
+
+<div class="release-feature-authors">authors: @mockersf</div>
+
+We take CI pretty seriously in Bevy land and we're always on the lookout for new ways to make our lives better. We made a number of nice improvements this cycle:
+
+* We now set an MSRV (minimum supported Rust version) for the `bevy` crate and we have a CI job that checks the MSRV
+* CI gives new contributors a friendly welcome message!
+* CI now asks for a migration guide when a PR is labeled as a breaking change and no migration guide is present
+
+## The First Subject Matter Expert Release
+
+This was our first release using our new [Subject Matter Expert (SME) system](/news/scaling-bevy-development/). We merged an absolutely massive amount of changes, and this was _despite_ our Project Lead `@cart` being away for about a month for Christmas and snowboarding vacations. We maintained a high quality bar and built amazing things. Suffice it to say the future is looking bright (and sustainable)! Stay tuned for more SME appointments in more areas.
+
+## What's Next?
```

Comment:
```suggestion
## <a name="what-s-next"></a>What's Next?
```
this anchor means that linking to this works even in non-sola rendered context. we need it to pass the markdown linter